pkg/astparser/parser.go

```diff
@@ -1491,6 +1491,8 @@ func (p *Parser) parseSelectionSet() (int, bool) {
 			}
 			ref := p.parseSelection()
 			set.SelectionRefs = append(set.SelectionRefs, ref)
+		case keyword.COMMENT:
+			p.read()
```

Comment:
Could you update the selection set test to show this works as expected? (Or, if you don't have the time let me know and I can help update the test.)